### ISSUE TITLE
Create Routes and Components for Navigation in admin dashboard

### ISSUE THREAD

**Issue by iamurmani** [iamurmani/skilluptechie#25]
## Objective
To design and implement routes and respective components for a web application that will allow users to navigate to various sections such as overview, analytics, questions, create questions, language, and create language.

## Components
1. **Overview**: Displays an overview of the application's main features.
2. **Analytics**: Displays analytical data and metrics.
3. **Questions**: Lists all the questions.
4. **Create Questions**: Allows the creation of new questions.
5. **Language**: Lists all the programming languages.
6. **Create Language**: Allows the creation of new languages.

**Comment by Madanraj0519** [comment]
I think you are saying that Just create a component according to the section and give a route to each section in the admin dashboard. thus it?

**Comment by iamurmani** [comment]
@Madanraj0519 yes because once if the component is ready, i can create separate tasks to work on the specific component design

**Comment by Madanraj0519** [comment]
I will take this issue

**Comment by chandankumarm55** [comment]
so basically u want to create a component for the above routes i.e  overview, analytics, questions, create questions, language, and create language. so i have one question is this component design  defined or  the data for implementing is coming from backend or we need to use dummy data . can u please tell about this more and also add some sample designs or sketches how the each component  should look like .
